streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorNodeTest.java

```diff
@@ -160,8 +160,8 @@ private void testMetrics(final String builtInMetricsVersion) {
     }
 
     @Test
-    public void testTopologyLevelClassCastException() {
-        // Serdes configuration is missing (default will be used which don't match the DSL below), which will trigger the new exception
+    public void testTopologyLevelConfigException() {
```

Comment:
It seems this test should verify the error for miss-configures (but existing serdes).

Thus, it seems we should set byte-array serdes on the config but not modify the test by any other means?

Reply:
I see what you mean, I think it would be valuable to have both. The catch for me was trying to figure out how to set streams configs in this test set up, is there a way to pass them in somewhere that I'm missing? I haven't been able to find anything @mjsax

Reply:
Nevermind, got it! It can be passed in with the topology test driver (: